### ISSUE TITLE
All methods are cached

### ISSUE THREAD

**Issue by paradoxxxzero** [lexich/redux-api#140]
PUT, POST, DELETE methods aren't executed when cache is enabled (and a previous request has been made within the cache window)

This is really problematic, only GET methods should be cached.
I think something like this should fix it:
```javascript
 const cacheManager = params.method === 'GET' && getCacheManager(options.expire, meta.cache);
```
https://github.com/lexich/redux-api/blob/master/src/actionFn.js#L59

Otherwise the cache works fine now (#135)

**Comment by lexich** [comment]
Yes, I think that you are right, but I don't think that fix should be so simple :)

**Comment by paradoxxxzero** [comment]
Well, I tried :)

On a side note I have another problem: I can't PUT 2 elements simultaneously due to this line https://github.com/lexich/redux-api/blob/master/src/actionFn.js#L132 the second gets ignored.
Is there any workaround?

**Comment by lexich** [comment]
You can't do this if you control state. But if you need put without update state (2 simultaneously request) you shoud use request method. It's simple request without redux wrappers

**Comment by paradoxxxzero** [comment]
Yes you're right thanks !

**Comment by ianloubser** [comment]
Having the same issue.
Has this been resolved? or is there a simple workaround?

**Comment by lexich** [comment]
@ianloubser use .request method and control state manually

**Comment by ianloubser** [comment]
Not for the multiple PUT requests, for the cached POST, PUT and DELETE methods.
The original issue described here
